src/ctia/lib/redis.clj

```diff
@@ -0,0 +1,31 @@
+(ns ctia.lib.redis
+  (:require [taoensso.carmine :as c]))
+
+;; Connections
+
+(defn server-connection
+  "Build the server config"
+  [[host port]]
+  (assert (and host port) "Redis has been de-configured")
+  {:pool {}
+   :spec {:host host
+          :port port
+          :timeout-ms 30000}})
+
+;; Pub/Sub
+
+(defn unsubscribe [pubsub-listener]
+  (c/with-open-listener pubsub-listener
+    (c/unsubscribe)))
+
+(defn subscribe [host-port event-channel-name listener-fn]
+  (c/with-new-pubsub-listener (:spec (server-connection host-port))
+    {event-channel-name listener-fn}
+    (c/subscribe event-channel-name)))
+
+(defn publish [host-port event-chanel-name event]
+  (c/wcar (server-connection host-port)
```

Comment:
Rebuilding the server-connection map each time we publish is inefficient, but I will resolve that in issue #255